operations/finance.md

```diff
@@ -347,80 +404,52 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `AccountingItems` | array of [Accounting item](finance.md#accounting-item) | optional | The accounting items. |
-| `CreditCardTransactions` | array of [Credit card transaction](finance.md#credit-card-transaction) | optional | The credit card payment transactions. |
+| `OrderItems` | array of [Order item](#order-item) | optional | The accounting items. |
+| `PaymentItems` | array of [Payment item](#payment-item) | optional | The accounting items. |
+| `CreditCardTransactions` | array of [Credit card transaction](#credit-card-transaction) | optional | The credit card payment transactions. |
 
-#### Accounting item
+#### Order item
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
 | `Id` | string | required | Unique identifier of the item. |
-| `CustomerId` | string | required | Unique identifier of the [Customer](customers.md#customer) whose account the item belongs to. |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the item belongs to. |
 | `OrderId` | string | optional | Unique identifier of the order \(or [Reservation](reservations.md#reservation) which is a special type of order\) the item belongs to. |
-| `ServiceId` | string | optional | Unique identifier of the [Service](services.md#service) the item belongs to. |
-| `ProductId` | string | optional | Unique identifier of the [Product](services.md#product). |
-| `BillId` | string | optional | Unique identifier of the bill the item is assigned to. |
-| `InvoiceId` | string | optional | Unique identifier of the invoiced [Bill](finance.md#bill) the item is receivable for. |
-| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](finance.md#accounting-category) the item belongs to. |
-| `CreditCardId` | string | optional | Unique identifier of the [Credit card](finance.md#credit-card) the item is associated to. |
-| `Type` | string [Accounting item type](finance.md#accounting-item-type) | required | Type of the item. |
-| `SubType` | string [Accounting item subtype](finance.md#accounting-item-subtype) | required | subtype of the item. Note that the subtype depends on the `Type` of the item.  |
-| `Name` | string | required | Name of the item. |
-| `Notes` | string | optional | Additional notes. |
-| `ConsumptionUtc` | string | required | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
+| `BillId` | string | optional | Unique identifier of the [Bill](#bill) the item is assigned to. |
+| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](#accounting-category) the item belongs to. |
+| `Amount` | [Amount value](#amount-value) | required | Item's amount, negative amount represents either rebate or a payment. |
```

Comment:
Shouldn't this be extended-amount?

Reply:
I'm not even sure this should be serialized as `ExtendedAmount` as `Value`, `Net`, `Tax` and `TaxRate` used to be part of old costs. cc @vitasamek

Reply:
It makes sense for the legacy `AccountingItems` collection, but not really for new ones.